debug-ui/scripts/run-rando-bot.ts

```diff
@@ -0,0 +1,603 @@
+import 'dotenv/config';
```

Comment:
this is all copy pasta from the the dummy random trader? I dont mind since dont foresee active development on it.

Reply:
yep. Very little changes to it.